### DISCUSSION TITLE
I cannot find open source code of gensim!!!

### DISCUSSION THREAD

**Discussion by arda1906** [community/community#123783] [Programming Help] [answered]
### Body

how can I find open sourcd code of gensim?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by nikhilsbuilds** [comment, marked as answer]
Gensim is an open-source Python library for topic modeling, document indexing, and similarity retrieval. Its source code is available on GitHub, which is a popular code hosting platform for open-source software projects.

To find the Gensim source code on GitHub, you can follow these steps:

1. Go to the Gensim GitHub repository: https://github.com/RaRe-Technologies/gensim

2. Once on the repository page, you can explore the source code by navigating through the directories and files.

3. If you want to download the entire source code, you can click on the green "Code" button and then select "Download ZIP" to download a ZIP file containing the entire repository.

4. Alternatively, you can clone the repository to your local machine using Git. Open your terminal or command prompt and navigate to the directory where you want to clone the repository. Then, run the following command:

```
git clone https://github.com/RaRe-Technologies/gensim.git
```

This will create a local copy of the Gensim repository on your machine, allowing you to explore and modify the source code as needed.

The Gensim repository on GitHub also includes additional resources like documentation, issue tracker, and contribution guidelines. You can explore these resources to better understand the project and potentially contribute to its development if you wish.

**Reply by arda1906** [reply]
thank you for your answer. but I wonder that how can I see other modules that there is not like word2vec but other modules  create the word2vec. gensim has them.🤔
